docs/reference/run.md

```diff
@@ -932,6 +932,16 @@ driver to a GELF remote server at `192.168.0.42` on port `12201`
 
 The `gelf-tag` option specifies a tag for easy container identification.
 
+#### Logging driver: fluentd
```

Comment:
This has some missing tagging and an awkward construction in the second option.

--> https://gist.github.com/moxiegirl/62fd3d0a52d6fb1d4853

#### Logging driver: fluentd

Fluentd logging driver for Docker. Writes log messages to `fluentd` (forward input). The `docker logs`
command is not available for this logging driver.

You can use the `--log-opt NAME=VALUE` flag to specify these additional Fluentd logging driver options.

 - `fluentd-address`: specify `host:port` to connect [localhost:24224]
 - `fluentd-tag`: specify tag for `fluentd` message, 

When specifying a `fluentd-tag` value, you can use the following markup tags:

 - `{{.ID}}`: short container id (12 characters)
 - `{{.FullID}}`: full container id
 - `{{.Name}}`: container name

For example, to specify both additional options:

`docker run --log-driver=fluentd --log-opt fluentd-address=localhost:24224 --log-opt fluentd-tag=docker.{{.Name}}`

If container cannot connect to the Fluentd daemon on the specified address, the container stops
immediately.